dcpy/lifecycle/ingest/templates/dpr_capitalprojects.yml

```diff
@@ -0,0 +1,66 @@
+id: dpr_capitalprojects
+acl: public-read
+
+attributes:
+  name: DPR Capital Projects
+  description: |
+    check [here](https://www.nycgovparks.org/bigapps/desc/DPR_CapitalProjectTracker_001.txt) for metadata
+  url: https://www.nycgovparks.org/bigapps
+
+ingestion:
+  source:
+    type: file_download
+    url: https://www.nycgovparks.org/bigapps/DPR_CapitalProjectTracker_001.json
+
+  file_format:
+    type: json
+    json_read_fn: normalize
+    json_read_kwargs:
+      {
+        "record_path": ["Locations", "Location"],
+        "meta": ["TrackerID", "FMSID", "Title", "TotalFunding"],
+      }
+    geometry:
+      crs: EPSG:4326
+      geom_column:
+        x: Longitude
+        y: Latitude
+
+  processing_steps:
+    - name: drop_columns
```

Comment:
Huh. There's an argument to just leave, I feel like there's no real downside to another descriptive column